src/feedback/components/AmountPicker.vue

```diff
@@ -12,58 +12,45 @@
     >
       {{ $t('ACTIVITY_FEEDBACK.SET_AMOUNT') }}
     </QBtn>
-    <div class="content">
-      <div class="row no-wrap">
-        <AmountBox
-          class="amount"
-          :amount="value"
-        />
+    <div class="content column no-wrap q-pl-lg q-pb-lg">
+      <div class="row no-wrap items-center justify-between q-pt-xs">
         <div
-          style="margin-left: .6em"
-          class="col"
-        >
-          <div
-            v-t="'ACTIVITY_FEEDBACK.AMOUNT'"
-            class="q-pb-sm"
-          />
-          <AmountViewer
-            v-if="!$q.platform.is.mobile"
-            :amount="value"
-          />
-        </div>
+          v-t="'ACTIVITY_FEEDBACK.AMOUNT'"
+          class="col-9 text-bold q-py-md"
+        />
         <QBtn
           v-if="!showOverlay"
           round
           flat
           color="red"
-          class="self-start"
+          class="self-center q-my-xs q-mr-sm"
           :title="$t('ACTIVITY_FEEDBACK.DELETE_AMOUNT')"
           @click="$emit('input', null)"
         >
           <QIcon name="fas fa-times" />
         </QBtn>
       </div>
-      <div class="row no-wrap">
-        <QSlider
-          :value="limitedValue"
-          :min="0"
-          :max="70"
-          :step="0.5"
-          label
-          class="self-center q-mx-md"
-          @input="$emit('input', arguments[0])"
-        />
-        <!-- don't use type="number" here because browsers might enforce different decimal setting
-        depending on browser locale-->
+      <div class="content-body row no-wrap items-center">
         <QInput
           v-model="valueToNumber"
-          size="4"
-          class="q-mr-md"
+          type="number"
+          inputmode="decimal"
+          min="0"
+          step="0.1"
+          class="q-pr-lg"
+          outlined
+          size="8"
         >
           <template #append>
             <span class="text-caption">kg</span>
           </template>
         </QInput>
+        <div
+          v-if="!$q.platform.is.mobile"
+          class="col q-ml-md"
+        >
+          <AmountViewer :amount="value" />
+        </div>
       </div>
```

Comment:
I wonder if the validation error message for that field could go somewhere in here?

It currently looks a little out of place and outside the grey box:

![errormessage](https://user-images.githubusercontent.com/31616/117693135-9ad85d80-b1b5-11eb-9bb8-a40ce26996c5.png)

Not very important though, as I think most people can input a number OK.

Reply:
yup, you are right. We'll fix this.

Reply:
It's better now. But not possible to have it inside the grey box as it's a general error-div in the parent FeedbackForm:
<img width="568" alt="Bildschirmfoto 2021-05-16 um 22 59 14" src="https://user-images.githubusercontent.com/23213583/118412597-290f8080-b69b-11eb-96e9-b21ea765ba52.png">

But I noticed errors in the console:
<img width="586" alt="Bildschirmfoto 2021-05-16 um 22 55 46" src="https://user-images.githubusercontent.com/23213583/118412656-80adec00-b69b-11eb-9a16-5b78accc1bc0.png">

I think we need to bring back the Number conversion in the set() of `valueToNumber`:
```js
set (v) {
  let value = parseFloat(v, 10)
  value = isNaN(value) ? 0 : Math.max(0, value)
  this.$emit('input', value)
},
```

The `isNaN` check will then prevent the error-div from showing up entirely.

Reply:
Number conversion is back. No more errors in console and the validation error message will not appear at all.